sphinx/builders/html.py

```diff
@@ -145,6 +153,12 @@ def init(self):
                 self.script_files.append('_static/translations.js')
         self.use_index = self.get_builder_config('use_index', 'html')
 
+        if self.config.html_experimental_html5_writer and not html5_ready:
+            self.app.warn(' '.join((
+                'html_experimental_html5_writer is set, but current version is old.',
+                'Docutils\' version should be or newer than 0.13.1, but %s.',
+            )) % __version_info__)
```

Comment:
Could you check is this work? `__version_info__` is a tuple containing version info (cf. `(0, 13, 1)`).
So python formatting might cause an error.